pkg/controller/command/verifiable/command.go

```diff
@@ -483,26 +516,39 @@ func (o *Command) createAndSignPresentationByID(vc *verifiable.Credential,
 
 func (o *Command) addLinkedDataProof(vp *verifiable.Presentation, opts *ProofOptions) (*verifiable.Presentation,
 	error) {
-	var s signer
+	var signer verifiablesigner.SignatureSuite
 
-	s, err := newKMSSigner(o.ctx.Signer(), o.kResolver, opts.VerificationMethod)
-	if err != nil {
-		return nil, err
+	signatureType := "Ed25519Signature2018"
+
+	// TODO  Need to provide the option to user for signature type. for now create jsonwebsignature2020
+	//  private key is not empty.
+	if opts.PrivateKey != "" {
```

Comment:
Can you create an issue to review if this option for passing private key should be permanent?

Reply:
https://github.com/hyperledger/aries-framework-go/issues/1682